public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -39,25 +38,25 @@ figure.image-display
 :marked
   We added two properties to the formerly empty component: `title` and `myHero`.
 
-  Our revised template displays the two component properties using the double curly brace
+  Our revised template displays the two component properties using double curly brace
   interpolation:
 
 +makeExample('displaying-data/ts/app/app.component.1.ts', 'template')(format=".")
 .l-sub-section
   :marked
-    The template is a multi-line string within ECMAScript 2015 back-tics (\`).
-    The back-tick (\`) is not the same character as a single quote (').
-    It has many nice features. The feature we're exploiting is
-    the ability to compose the string over several lines which
+    The template is a multi-line string within ECMAScript 2015 backticks (\`).
+    The backtick (\`)—which is *not* the same character as a single quote (')—has
```

Comment:
I recommend replacing the *hyphen-without-spaces* by `&mdash;`  here and everywhere in this document.